### PR TITLE
CPack NSIS Installer.

### PR DESCRIPTION
**Pull request by AYESDIE**
Provides CPack configuration to make a Windows NSIS Installer.

image

image

image

### CODE REVIEW

**Comment by AYESDIE** [comment]
@morevnaproject Are there any Synfig banners that can be used with NSIS installer? I couldn't find one so I made *HEADER* and *WELCOMEFINISH* banners myself. 

HEADER


WELCOMEFINISH

**Comment by morevnaproject** [comment]
There is "installer_logo.sif" file in "synfig-studio/images" dir.
But they are dated from 2007 and ones you made are look better ^__^!

File: cmake/InstallMSYS2.cmake [hunk 1]
```diff
@@ -82,6 +83,7 @@ if(WIN32)
             ERROR_QUIET
             OUTPUT_STRIP_TRAILING_WHITESPACE
         )
+        string(REPLACE "\\" "/" MLT_DIRECTORY "${MLT_DIRECTORY}")
```
Comment (ice0) [inline cmake/InstallMSYS2.cmake:86]: You can use `cygpath -m <path>` and get Windows paths with slashes, and you won't need to replace them..

**Comment by ice0** [comment]
Merged. Thank you!

**Pull request merged**